layouts/dashboard.vue

```diff
@@ -1,59 +1,78 @@
 <template>
     <div>
-        <app-header />
-        <dashboard-header />
+        <AppHeader />
+        <DashboardHeader />
         <Nuxt />
-        <kyb-alert v-if="showKybAlert" />
-        <kyc-alert v-if="showKycAlert" />
-        <b-alert
+        <KYBAlert v-if="showKybAlert" />
+        <KYCAlert v-if="showKycAlert" />
+        <BAlert
             id="verify-mobile"
             :show="showVerifyMobileAlert && !showVerifyEmailAlert"
             class="fixed-bottom bottom-left-alert m-4 p-4"
             variant="bg-colored"
         >
             We need to verify your mobile number. Please click
             <b-link class="link" to="/login/verify/mobile"> here.</b-link>
-        </b-alert>
-        <b-alert
+        </BAlert>
+        <BAlert
             id="verify-email"
             :show="showVerifyEmailAlert"
             class="fixed-bottom bottom-left-alert m-4 p-4"
             variant="bg-colored"
         >
             We need to verify your email address. Please click
             <b-button class="link mb-1" variant="transparent" @click="goToVerify">here.</b-button>
-        </b-alert>
+        </BAlert>
         <LoadingSpinner id="loader" :is-loading="isLoading" />
-        <cookie-policy />
+        <Cookie />
     </div>
 </template>
 
 <script lang="ts">
```

Comment:
Minor but setup can be used here because page meta is not defined. It works this way still.

Reply:
refactored to setup sugar